CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- `tt replicaset promote`: command to promote an instance in the tarantool 3.0 replicaset.
+- `tt replicaset config_source`: module to manage replicaset via 3.0 cluster config source.
```

Comment:
Agree, but same time I didn't use an option because it can be confusing. For example, `-f` has different meaning. the URI format is a little different and `orchestrator` (what if the others flags will be need?) has not any sense for the cconfig source.
How do you think, it should be an option?

Reply:
I think it's ok like a subcomand, but I don't like the name.